test/e2e/e2e_test.go

```diff
@@ -268,13 +267,20 @@ func TestNewApplicationVanguard(t *testing.T) {
 	}
 }
 
-func TestRolloutVanguard(t *testing.T) {
+func TestNewApplicationVanguardMultipleReplicas(t *testing.T) {
+	testNewApplicationVanguard(4, t)
```

Comment:
Also probably worth putting a note as to why we have these distinct cases (1 pod vs 4 pods).